### PR TITLE
Implement default API key

### LINKED ISSUE #70: There is no need for API key request from user
https://github.com/jellyfin/jellyfin-docs/blob/master/general/server/plugins/open-subtitles.md/#L1

```
This plugin will allow your server to download subtitles from OpenSubtitles.com for any video file on your server. The plugin can be installed from the catalog page and once enabled you will need to enter your OpenSubtitles.com account info and an API key (which can be obtained here) in the plugin configuration page.
```

Thing is, there is no need to ask for API key. API key should be provided within your Program (you get it from your profile and name it as Jellyfin for example), we just need to have user credentials. We are doing statistics per User Agent and now there is many jellyfin API keys (consumers)

### PR DESCRIPTION
**Pull request by MBR-0001**
Part 2/2 of changes suggested/requested by OpenSubtitles

Adds the default API key which the plugin uses if the user hasn't provided their own

Closes #70

draft until the official jellyfin API key is provided

### CODE REVIEW

**Comment by crobibero** [comment]
@jellyfin-bot rebase

**Comment by MBR-0001** [comment]
@crobibero I tested the functionality and it works correctly, any chance this and #83 could be done in a single version (v16)?

**Comment by crobibero** [comment]
> @crobibero I tested the functionality and it works correctly, any chance this and #83 could be done in a single version (v16)?

Will hold off on creating release until both are merged. Hopefully will have time to review tonight after work

**Review by crobibero — APPROVED** [review]

**Pull request merged**
